DigitalExcellence/dex-frontend
# Redesign homepage

## Description

Implementation of the new homepage designs

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to the homepage
2. Check if all buttons redirect to the expected page
3. Check if everything looks good on all resolutions
4. Check if the modals open on the homepage and work as expected

## Link to issue

Closes: #380
